lib/client.js

```diff
@@ -38,11 +39,38 @@ class Metacom extends EventEmitter {
     this.api = {};
     this.callId = 0;
     this.calls = new Map();
+    this.streamId = 0;
+    this.streams = new Map();
     this.socket.addEventListener('message', ({ data }) => {
-      try {
-        const packet = JSON.parse(data);
-        const { callback, event } = packet;
-        const callId = callback || event;
+      if (typeof data === 'string') void this.message(data);
+      else void this.binary(data);
```

Comment:
void again?

Reply:
Yes, just to explicitly dispose the returning promise and make it intentional.